src/components/sequencing/CommandPanel/SelectedCommand.svelte

```diff
@@ -0,0 +1,228 @@
+  }
+
+  function onSelectCommandDefinition() {
+    dispatch('selectCommandDefinition', commandDef);
+  }
+</script>
+
+<div id={ID_COMMAND_DETAIL_PANE} class="content">
+  {#if commandName != null}
+    <div class="command-name">
+      {commandName}
+      <button class="open-dictionary" on:click={onSelectCommandDefinition}><ArrowUpRightIcon /></button>
+    </div>
+  {/if}
+  {#if commandDef != null && commandDef.description != null}
+    <div class="command-description">{commandDef.description}</div>
+  {/if}
+  {#if !!timeTagNode}
+    <fieldset>
+      <label class="label-row" for="timeTag">Time Tag</label>
+      <input class="st-input w-100" disabled name="timeTag" value={timeTagNode.text.trim()} />
+    </fieldset>
+  {/if}
+  {#if !!commandNode}
+    {#if commandInfoMapper.nodeTypeHasArguments(commandNode)}
+      {#if !!commandDef}
+        <fieldset>
+          <Collapse headerHeight={24} title={commandDef.stem} padContent={false}>{commandDef.description}</Collapse>
+        </fieldset>
+
+        {#each editorArgInfoArray as argInfo}
+          <ArgEditor
+            {argInfo}
+            {commandDictionary}
+            {commandInfoMapper}
+            {variablesInScope}
+            setInEditor={debounce((token, val) => setInEditor(editorSequenceView, token, val), 250)}
+            addDefaultArgs={(commandNode, missingArgDefArray) =>
+              addDefaultArgs(commandDictionary, editorSequenceView, commandNode, missingArgDefArray, commandInfoMapper)}
+          />
+        {/each}
+
+        {#if missingArgDefArray.length}
+          <fieldset>
+            <AddMissingArgsButton
+              setInEditor={() => {
+                if (commandNode) {
+                  addDefaultArgs(
+                    commandDictionary,
+                    editorSequenceView,
+                    commandNode,
+                    missingArgDefArray,
+                    commandInfoMapper,
+                  );
+                }
+              }}
+            />
+          </fieldset>
+        {/if}
+      {:else}
+        <fieldset>
+          <div class="label-row">{commandName ?? ''}</div>
+        </fieldset>
+        <div class="empty-state st-typography-label">Command type is not present in dictionary</div>
+      {/if}
+    {:else}
+      <fieldset>
+        <div class="label-row">{`${formatTypeName(commandNode.name)} Name`}</div>
+        <div>
+          <StringEditor
+            argDef={nameArgumentDef}
+            initVal={commandName ?? ''}
+            setInEditor={val => {
+              if (commandNameNode) {
+                setInEditor(editorSequenceView, commandNameNode, val);
+              }
+            }}
+          />
+        </div>
+      </fieldset>
+    {/if}
+  {:else}
+    <div class="empty-state st-typography-label">
+      Select a command or open the
+      <div class="no-wrap">
+        <button class="st-button-link" on:click={onSelectCommandDefinition}> command dictionary </button>.
```

Comment:
Can this get some typography class like label or body?

Reply:
Can we also add it to the other `st-button-link` instances in the *results?

Reply:
Or consider just adding label or body to `.dictionary-results` so that both the results and the empty state messages can have type styles?